src/node.cc

```diff
@@ -4244,11 +4249,15 @@ static void StartNodeInstance(void* arg) {
       SealHandleScope seal(isolate);
       bool more;
       do {
+#ifndef NODE_EMBEDDED_MODE
         v8::platform::PumpMessageLoop(default_platform, isolate);
+#endif  // NODE_EMBEDDED_MODE
         more = uv_run(env->event_loop(), UV_RUN_ONCE);
 
         if (more == false) {
+#ifndef NODE_EMBEDDED_MODE
           v8::platform::PumpMessageLoop(default_platform, isolate);
```

Comment:
Out of curiosity, how are embedders supposed to be able to perform these same tasks with node's main event loop?